libtorch-ffi/libtorch-ffi.cabal

```diff
@@ -65,17 +65,17 @@ library
                     , async
  extra-libraries:     stdc++
                     , c10
-                    , iomp5
-                    , mklml
+                    -- , iomp5
```

Comment:
this can be uncommented if mkl is linked.

Reply:
ref: https://github.com/hasktorch/hasktorch/blob/4019d42039c4ec5f519530df46c9aa36a38fa46f/libtorch-ffi/default.nix#L14

Reply:
@wavewave have a look now, I uncommented `iomp5` and told nix to fill it in with `mkl` ;)

Reply:
great! thanks.